### DISCUSSION TITLE
“Create contact” flow on "Add to contacts"

### DISCUSSION THREAD

**Discussion by rapidddenis** [status-im/status-legacy#12626] [Ideas]
### User i Story

As a user I want to be promted with “create contact” flow when pressing “add to contacts”. 

### Description

on “add to contacts”:
0. open profile, 
1. set nickname and privacy settings
2. press “done”

provide default values, for example, nickname and image if shared by contact

![2C547164-1E80-4A15-9492-526C14C05D01](https://user-images.githubusercontent.com/41779817/130596606-0cc97ee3-91d4-4613-ba17-1bd0c9ddb3f2.png)

**Comment by churik** [comment]
Not sure if it makes sense now with https://github.com/status-im/status-react/issues/12175, converting to idea for now
